packages/gatsby-cli/src/reporter/redux/internal-actions.ts

```diff
@@ -286,6 +286,26 @@ export const endActivity = ({
     }
   }
 
+  // Show message if they haven't already enabled the feature and the js build
+  // duration is greater than 20 seconds (much longer than normal.
+  if (
```

Comment:
We'll probably want two tweaks:

1. Show this a max of one time ever for a given user / machine? (configstore?)
2. Do not show this in CI (granted, edge case, so take it or leave it)

Reply:
@pieh used to fix this stuff with (aka for) me -- I'm sure he has some ideas to show it only once 👍

Reply:
I actually didn't implement it yet, but for not showing too often I want to use something like we have in `feedback` survey: https://github.com/gatsbyjs/gatsby/blob/773b5f403ba971bcab4e9f75014fcee099bb2572/packages/gatsby/src/utils/feedback.ts#L100-L111
and for CI check just `isCI()` from `gatsby-core-utils`

Reply:
https://github.com/gatsbyjs/gatsby/pull/28120 is the util and in the description I linked commit with how I use it for query on demand